SP_aircraft.py

```diff
@@ -345,33 +345,33 @@ def run_optimal_737(objective, fixedBPR, pRatOpt = False):
     
     substitutions = get737_optimal_subs()
 
-    if Nmission > 1:
+    if Nmission == 2:
         substitutions.update({
-            'R_{req}': [3000.*units('nmi'),2000.*units('nmi')],
-            'n_{pass}': [180., 180.],
+            'R_{req}': [3000.*units('nmi'),3500.*units('nmi')], #,2500.*units('nmi')
+            'n_{pass}': [180., 160.], #, 140.
         })
     else:
         substitutions.update({
-##           'n_{paxx}': 180.,
-            'R_{req}': 3000.*units('nmi'),
-        })        
+           'R_{req}': 3000.*units('nmi'),
+           'n_{pass}': 180.,
+        })
 
     if fixedBPR:
         substitutions.update({
             '\\alpha_{max}': 6.97,
         })
         
     if pRatOpt:
-        del substitutions['\pi_{f_D}']
-##        del substitutions['\pi_{lc_D}']
-##        del substitutions['\pi_{hc_D}']
+        #del substitutions['\pi_{f_D}']
+        del substitutions['\pi_{lc_D}']
+        del substitutions['\pi_{hc_D}']
 
```

Comment:
so now pRatOpt allows no fan optimization but does allow LPC and HPC optimization?

Reply:
I'm working on making it totally general, but for now no.